src/CategoricalArrays.jl

```diff
@@ -1,14 +1,14 @@
 __precompile__()
 module CategoricalArrays
-    export CategoricalPool, CategoricalValue
+    export CategoricalPool, CategoricalValue, CategoricalString
     export AbstractCategoricalArray, AbstractCategoricalVector, AbstractCategoricalMatrix,
            CategoricalArray, CategoricalVector, CategoricalMatrix
     export AbstractNullableCategoricalArray, AbstractNullableCategoricalVector,
            AbstractNullableCategoricalMatrix,
            NullableCategoricalArray, NullableCategoricalVector, NullableCategoricalMatrix
     export LevelsException
 
-    export categorical, compress, decompress, droplevels!, levels, levels!, isordered, ordered!
+    export categorical, compress, decompress, droplevels!, levels, levels!, isordered, ordered!, iscatvalue
```

Comment:
Better not export this for now as `CatValue` isn't supposed to be extended as long as it's not a real trait.